content/en/_index.html

```diff
@@ -24,8 +24,8 @@
 {{% /blocks/feature %}}
 
 
-{{% blocks/feature icon="fab fa-github" title="Contributions welcome!" url="https://github.com/apache/parquet-mr" %}}
-We do a [Pull Request](https://github.com/apache/parquet-mr/pulls) contributions workflow on **GitHub**. New users are always welcome!
+{{% blocks/feature icon="fab fa-github" title="Contributions welcome!" url="https://github.com/apache/parquet-java" %}}
```

Comment:
As part of another PR perhaps we should revisit this link (perhaps it should link to parquet-format?) as again linking to the java implementation from the homepage might be more confusing than helpful

Reply:
That's a great point. This was actually that was torn about the most when I first built the new site. I figured more people would want to contribute to `mr` than `format` (there's actual code in the former), so I went with `mr` everywhere. More than happy to revisit this as it was mostly just a guess on my part.

Reply:
I think it is something to revisit as a follow on PR